src/bioutils/sequences.py

```diff
@@ -441,7 +453,12 @@ def translate_cds(seq, full_codons=True, ter_symbol="*"):
     protein_seq = list()
     for i in range(0, len(seq) - len(seq) % 3, 3):
         try:
-            aa = dna_to_aa1_lut[seq[i:i + 3]]
+            the_seq = seq[i:i + 3]
```

Comment:
Can we rename `the_seq` to `codon` for clarity?

Reply:
sure thing!